### PR TITLE
media-libs/libharu: add vtk compat patch; sci-libs/vtk: version bump to 8.1.0

### PR DESCRIPTION
**Pull request by dracwyrm**
**media-libs/libharu: Add vtk compat patch**
VTK 8.1.0 and above need support for free form triangle shading.
This patch helps prevent having two different copies of libharu
installed on a system

**sci-libs/vtk: version bump to 8.1.0**
Closes: https://bugs.gentoo.org/638024

@SoapGentoo @Amynka 
Could use some testing. :)

### CODE REVIEW

**Comment by dracwyrm** [comment]
@SoapGentoo @Amynka Other distros have added the libharu patch on their systems. This will prevent multiple copies of the program on a system, thus having the potential to mess things up. This is the most elegant solution to the compile issue.

**Comment by dracwyrm** [comment]
I'm updating the libharu ebuild, just pushed initial mod so patch can be evaluated while the ebuild undergoes reconstruction.

**Comment by dracwyrm** [comment]
This should close https://bugs.gentoo.org/596578 but that will need testing.

File: media-libs/libharu/libharu-2.3.0-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+AUTOTOOLS_AUTORECONF=1
```
Comment (SoapGentoo) [inline media-libs/libharu/libharu-2.3.0-r2.ebuild]: remove

File: media-libs/libharu/libharu-2.3.0-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+SRC_URI="https://github.com/${PN}/${PN}/archive/${MYP}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+DEPEND="
+	>=media-libs/libpng-1.2.51[${MULTILIB_USEDEP}]
+	>=sys-libs/zlib-1.2.8-r1[${MULTILIB_USEDEP}]"
```
Comment (SoapGentoo) [inline media-libs/libharu/libharu-2.3.0-r2.ebuild]: both need `:=`, and remove version specifiers on both

**Pull request closed without merging**